package.xml

```diff
@@ -37,6 +37,7 @@
   <build_depend>message_generation</build_depend>
   <build_depend>python-catkin-pkg</build_depend>
   <build_depend>roslint</build_depend>
+  <build_depend>geographiclib</build_depend>
```

Comment:
I'd like to retain the alphabetical order of these, if possible.

Reply:
Done.